protobuf/src/text_format/print.rs

```diff
@@ -5,12 +5,14 @@ use crate::core::Message;
 use crate::reflect::ReflectFieldRef;
 use crate::reflect::ReflectValueRef;
 
-fn quote_bytes_to(bytes: &[u8], buf: &mut String) {
+#[doc(hidden)]
+pub fn quote_bytes_to(bytes: &[u8], buf: &mut String) {
     for &c in bytes {
         match c {
             b'\n' => buf.push_str(r"\n"),
             b'\r' => buf.push_str(r"\r"),
             b'\t' => buf.push_str(r"\t"),
+            b'\'' => buf.push_str("\\\'"),
```

Comment:
Same here. What Google does in it's protobuf impl?

Can you please submit a separate PR with just formatting changes?

Reply:
Sure, rebased out all but the test itself.